### PR TITLE
Fix issue #199

### PR DESCRIPTION
**Pull request by danizen**
- Makes `LoginView` and `LogoutView` more consistent with `django.contrib.auth`
- Addresses my use case
- Includes tests

### CODE REVIEW

**Comment by danizen** [comment]
I got different results running isort outside of tox, because my virtualenv makes django-cas-ng *the* local package.   Sorry, will fix.

**Comment by danizen** [comment]
@mingchen, this should be ready now to merge once you take a look.  You can see my notes about the troubles I had with isort in issue #199.

File: django_cas_ng/views.py [hunk 1]
```diff
@@ -49,23 +49,27 @@ def successful_login(self, request, next_page):
         """
         return HttpResponseRedirect(next_page)
 
-    def post(self, request):
+    def post(self, request, *args, **kwargs):
         if request.POST.get('logoutRequest'):
-            next_page = request.POST.get('next', settings.CAS_REDIRECT_URL)
+            next_page = kwargs.get('next_page', None)
```
Comment (mingchen) [inline django_cas_ng/views.py]: Could you document this new parameter to README related section ?

**Comment by duoi** [comment]
It appears that I had set everything on fire at some point in #179 by not pushing a commit that made it backwards compatible, unfortunately it was missed in review.

Since we're already here, would you mind, as part of your PR, be able to take a look at this? The required changes are, from memory, just a matter of exposing the following lines at the end of `views.py`.

```
login = LoginView.as_view()
logout = LogoutView.as_view()
callback = CallbackView.as_view()
```

This makes `cas_ng_views.login`,  `cas_ng_views.logout` and  `cas_ng_views.callback` importable and work as normal.

**Comment by danizen** [comment]
> This makes `cas_ng_views.login`, `cas_ng_views.logout` and `cas_ng_views.callback` importable and work as normal.

Very important.  It caused 5-10 applications at the National Library of Medicine to break, but not solely due to this error; it took my lack of care as well.  I thought I had been quite careful in testing my new version of a private wrapper.  However, the moment I made 3.6.0 available in our private repository, users whose requirements files listed something like "django_cas_ng>=3.5.7" and my private wrapper fixed "nlm-occs=1.2.0", well, no one could log into those applications.

Anyway, hard to be perfect, but it was a fire when it happened.

**Comment by mingchen** [comment]
Please resolve conflict.

**Pull request closed without merging**